x-pack/plugins/security_solution/public/graphql/types.ts

```diff
@@ -1070,6 +1070,8 @@ export interface RuleField {
 
   note?: Maybe<string[]>;
 
+  threshold?: Maybe<ToAny>;
+
```

Comment:
Where is the source declaration that results in these generated types for timeline? Looks like I need to add the new rule fields from https://github.com/elastic/kibana/pull/70288. This ensures these fields show up in the field browser for timeline, correct?